### PR TITLE
fix styling issues

### PR DESCRIPTION
**Pull request by vmatsiiako**
# Description 📣

## Type ✨

- [x] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Documentation

- [x] I have read the [contributing guide](https://infisical.com/docs/contributing/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/code-of-conduct). 📝

### CODE REVIEW

File: frontend/src/layouts/AppLayout/AppLayout.tsx [hunk 1]
```diff
@@ -143,6 +146,23 @@ export const AppLayout = ({ children }: LayoutProps) => {
 
   const { t } = useTranslation();
 
+  const registerUserAction = useRegisterUserAction();
+
+  useEffect(async () => {
```
Comment (akhilmhdh) [inline frontend/src/layouts/AppLayout/AppLayout.tsx:151]: Put a dependency or even empty one. This gets fired on each render

Comment (akhilmhdh) [reply frontend/src/layouts/AppLayout/AppLayout.tsx:151]: Also why not move this also to react-query.

**Review by akhilmhdh — CHANGES_REQUESTED** [review]

File: frontend/src/layouts/AppLayout/AppLayout.tsx (hunk 1 above)
Comment (dangtony98) [reply frontend/src/layouts/AppLayout/AppLayout.tsx:151]: Agree this should be moved to `react-query`; I'll make the adjustment.

Comment (dangtony98) [reply frontend/src/layouts/AppLayout/AppLayout.tsx]: Updated.

**Pull request merged**